install_config/web_console_customization.adoc

```diff
@@ -112,7 +109,29 @@ window.OPENSHIFT_EXTENSION_PROPERTIES = {
   key2: "value2",
 }
 ----
-====
+
+[[extension-option-for-external-logging-solutions]]
+=== Extension Option for External Logging Solutions
+
+As of {product-title} 3.6, there is an extension option to link to external
+logging solutions instead of using {product-title}'s EFK logging stack:
+
+----
+'use strict';
+angular.module("mylinkextensions", ['openshiftConsole'])
+       .run(function(extensionRegistry) {
+          extensionRegistry.add('log-links', _.spread(function(resource, options) {
+            return {
+              type: 'dom',
+              node: '<span><a href="https://extension-point.example.com">' + resource.metadata.name + '</a><span class="action-divider">|</span></span>' <1>
+            };
+          }));
+       });
+hawtioPluginLoader.addModule("mylinkextensions");
+----
+<1> The extension file (`\https://extension-point.example.com`, in this example)
```

Comment:
"The extension file" doesn't make sense to me here.  Its the URL to the logging stack you are wanting to access.